src/css/application.css

```diff
@@ -604,3 +604,21 @@ body {
 .u__hidden {
   display: none;
 }
+
+.loading {
+  align-items: center;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  width: 100%;
+}
+
+.loading__spinner {
+  background-image: url('/images/spinner.gif');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  cursor: not-allowed;
+  height: 100px;
+  width: 100px;
+}
```

Comment:
Done! I don't usually set that up in favor of following local linter rules. Any feelings on adding something line `css-lint` (which has https://github.com/gajus/css-lint/blob/master/docs/rules/final-newline.md)?